app/screens/more_channels/more_channels.js

```diff
@@ -362,8 +396,24 @@ export default class MoreChannels extends PureComponent {
                             value={term}
                         />
                     </View>
+                    <Picker
+                        selectedValue={this.state.typeOfChannels}
+                        style={{height: 50, width: 250}}
+                        onValueChange={(itemValue) =>
+                            this.setState({typeOfChannels: itemValue})
+                        }
+                    >
+                        <Picker.Item
+                            label='Show: Public Channels'
```

Comment:
all labels should be translatable. Also `Show` should be part of the title of the bottom/action sheet, that is why the picker is not the right component.